### PR TITLE
346 table column emptiness

### PR DESCRIPTION
**Pull request by sampacos**
Add methods and tests for verifying table column is completely empty or not empty #346

### CODE REVIEW

File: src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java [hunk 1]
```diff
@@ -220,4 +220,33 @@ public static void verifyCellInSpecifiedRow(String rowNum, String columnName, St
...
+
+        var expectedResult = SentinelStringUtils.format(
+                "Expected all cells in the {} column of the {} to be {}empty.",
+                columnName, tableName, (negate ? "not " : ""));
+        log.trace(expectedResult);
+
+        if (negate) {
+            assertTrue(expectedResult, getElementAsTable(tableName).verifyAllColumnCellsNotEmpty(columnName));
+        } else {
+            assertTrue(expectedResult, getElementAsTable(tableName).verifyAllColumnCellsEmpty(columnName));
```
Comment (dougnoel) [inline src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java:248]: I don't see a test for this. Is this code covered?

**Review by dougnoel — CHANGES_REQUESTED** [review]

**Review by dougnoel — APPROVED** [review]
